chirpy/api.py

```diff
@@ -195,6 +195,9 @@ def __call__(self, **kwargs):
         elif method == 'POST':
             post_data = json.dumps(json_body) if json_body is not None else kwargs
             request = partial(requests.request, data=post_data)
+        elif method == 'PUT':
+            put_data = json.dumps(json_body) if json_body is not None else kwargs
+            request = partial(requests.request, params=kwargs, data=put_data)
```

Comment:
Generally only GET requests have query params. Note the requests docs for [get](http://docs.python-requests.org/en/master/api/#requests.get) and [put](http://docs.python-requests.org/en/master/api/#requests.put) say as much. That said, REST is whatever you want it to be, so if this works for your use case then it's ok with me.

Reply:
Yeah, ideally I wouldn't have that bit, but for the endpoint I need this for, Twitter uses querystring params with a PUT request https://developer.twitter.com/en/docs/direct-messages/welcome-messages/api-reference/update-welcome-message